framework/assets/yii.js

```diff
@@ -329,7 +329,7 @@ yii = (function ($) {
             }
 
             if (message !== undefined) {
-                pub.confirm(message, function () {
+                $.proxy(pub.confirm, this)(message, function () {
                     pub.handleAction($this, event);
```

Comment:
@lichunqiang you show the closure signature `function(message, ok, cancel)`, but I can't figure out  where the `cancel` attribute will be passed from.